cloudify_cli/commands/nodes.py

```diff
@@ -87,6 +90,21 @@ def get(node_id, deployment_id, logger, client, tenant_name):
         logger.info('\t{0}: {1}'.format(property_name, property_value))
     logger.info('')
 
+    if get_global_verbosity() != NO_VERBOSE:
+        operations = []
+        for op in utils.decode_dict(
```

Comment:
If working with `iteritems` it's often clearer to do:
```
for op_name, op in utils.decode_dict(node.operations).iteritems():
    op['name'] = op_name
    operations += op
```

Reply:
fixed, just small update:
  operations += [op]